### PR TITLE
Update traefik to v1.0.0-rc1

### PR DESCRIPTION
**Pull request by emilevauge**
Hello,
This PR updates traefik to `v1.0.0-rc1` 
/cc @vdemeester
Cheers :)
Emile

Signed-off-by: Emile Vauge <emile@vauge.com>

### CODE REVIEW

**Comment by vdemeester** [comment]
:tada:

**Comment by samber** [comment]
season-4-the-simpsons-4x2-l2JdVRfJozpjq70SA

**Comment by tianon** [comment]
LGTM (https://github.com/containous/traefik-library-image/compare/0b0c26da8cb7abc1190d95acbfc5c5e1f986f685...7eaf03d205ba9bbd6dbcad2675516c33043f79ff)

Build test of #1789; 09d57e6 (`traefik`):

```console
$ bashbrew build "traefik"
Cloning traefik (git://github.com/containous/traefik-library-image) ...
Processing traefik:v1.0.0-rc1 ...
Processing traefik:v1.0.0-beta.704 ...
Processing traefik:v1.0.0-beta.404 ...
Processing traefik:v1.0.0-beta.392 ...
Processing traefik:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing traefik:v1.0.0-rc1
	'cve-2014--shellshock' [1/1]...passed
testing traefik:v1.0.0-beta.704
	'cve-2014--shellshock' [1/1]...passed
testing traefik:v1.0.0-beta.404
	'cve-2014--shellshock' [1/1]...passed
testing traefik:v1.0.0-beta.392
	'cve-2014--shellshock' [1/1]...passed
```

File: library/traefik [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # maintainer: Emile Vauge <emile@vauge.com> (@emilevauge)
 # maintainer: Vincent Demeester <vincent@sbr.pm> (@vdemeester)
 
+v1.0.0-rc1: git://github.com/containous/traefik-library-image@7eaf03d205ba9bbd6dbcad2675516c33043f79ff
 v1.0.0-beta.704: git://github.com/containous/traefik-library-image@0b0c26da8cb7abc1190d95acbfc5c5e1f986f685
 v1.0.0-beta.404: git://github.com/containous/traefik-library-image@04d09c3cf006f360611fcb178d8537c318100d85
 v1.0.0-beta.392: git://github.com/containous/traefik-library-image@7f7b1e2c7d8ca00ae07e37e64a0e305f7422f6aa
```
Comment (tianon) [inline library/traefik]: Any particular reason to keep these older versions listed here?  Removing them here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (vdemeester) [inline library/traefik]: hum yeah, I think we could remove them, @emilevauge wdyt ?

Comment (emilevauge) [inline library/traefik]: I will clean old releases in the next PR ;)

Comment (tianon) [inline library/traefik]: Sounds good, thanks!

**Pull request merged**